.buildkite/pipeline.yml

```
@@ -0,0 +1,51 @@
+# Nodes with values to reuse in the pipeline.
+common_params:
+  plugins: &common_plugins
+  - &bash_cache automattic/bash-cache#v1.5.0: ~
+  env: &common_env
+    IMAGE_ID: xcode-12.5.1
+
+# This is the default pipeline – it will build and test the pod
+steps:
+  #################
+  # Build and Test
+  #################
+  - label: "🧪 Build and Test"
+    key: "test"
+    command: "build_and_test_pod"
+    env: *common_env
+    plugins: *common_plugins
+
+  #################
+  # Validate Podspec
+  #################
+  - label: "🔬 Validating Podspec"
```

Nitpick:

```suggestion
  - label: "🔬 Validate Podspec"
```

As per [this comment](https://github.com/wordpress-mobile/WordPress-iOS-Shared/pull/297#discussion_r693627008) on a similar PR.